.github/workflows/CI.yml

```diff
@@ -40,8 +40,11 @@ jobs:
         run: |
           cd adoc
           make OUTDIR=/tmp/out QUIET= html pdf
+      - name: Verify reflow conformance
+        run: |
+          ./adoc/scripts/verify_reflow_conformance.sh
```

Comment:
If I'm reading the script right, it's requiring that running reflow does nothing? That seems likely to make essentially every PR fail CI until you condition people to do this whenever they make a spec update. I've found it easier as the Vulkan spec editor to just do this when integrating PRs.

Reply:
>If I'm reading the script right, it's requiring that running reflow does nothing?

Yes. If I remember correctly, it was an ask from @gmlueck .  

It looks like GitHub support issuing [warning](https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions#setting-a-warning-message) if we want to go this way.